pkg/crc/machine/bundle/repository.go

```diff
@@ -97,7 +95,7 @@ func (repo *Repository) Use(bundleName string) (*CrcBundleInfo, error) {
 
 func (bundle *CrcBundleInfo) copyExecutableFromBundle(destDir string, fileType FileType, executableName string) error {
 	srcPath := bundle.getHelperPath(fileType)
-	if srcPath == "" {
+	if srcPath == "" || !crcos.FileExists(srcPath) {
```

Comment:
I don't think the `crcos.FileExists(srcPath)` check is needed, if the file is listed in the metadata, then `verify()` checked that the file exists. I think initially we did not have the preset checks, and we relied on `srcPath==""` to do the right thing for podman/openshift bundles. Then we added the IsPodman/IsOpenShift checks, but we kept the`srcPath==""` checks anyway, even if they are redundant.